ci/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+yarn install --no-progress
+yarn test
```

Comment:
Re: prettier failing, not sure... I can look into it

instead of running just `test`, can we run `preversion` on every FE lib? it is sort of a convention to punt `test` `dtslint` `prettier` and whatever other task we want to run before releasing a new version, which is ok to be run on every commit on the CI I think

Reply:
Prettier failing, but only on the CI, that's the weird thing...did it happen already on other projects?

Reply:
For now I update to `preversion` here, we can do it opportunistically on other libs since they were all running test right now...